jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Switch.java

```diff
@@ -61,6 +65,114 @@ public boolean isEnabled() {
         return !hasClass("v-input--is-disabled");
     }
 
+    @JDIAction("Get {name}'s color")
+    public String getColor() {
+        return css("color");
+    }
+
+    @JDIAction("Get {name}'s background color")
+    public String hasBackgroundColor() {
+        return css("background-color");
+    }
+
+    @JDIAction("Get {name}'s slot background color")
+    public String hasSlotsBackgroundColor() {
+        return core().find(".v-input__slot").css("background-color");
+    }
+
+    @JDIAction("Check that {name} has messages")
+    public boolean hasMessages() {
+        return core().finds(".v-messages__message").size() > 0;
+    }
+
+    @JDIAction("Check that {name} has error message")
+    public boolean hasErrorMessage() {
+        return hasClass("error--text");
+    }
+
+    @JDIAction("Check that {name} has success message")
+    public boolean hasSuccessMessage() {
+        return hasClass("success--text");
+    }
+
+    @JDIAction("Get {name}'s messages")
+    public String getMessage() {
```

Comment:
What do you think about combine getMessage() and getMessages()?

Reply:
I cannot imagine it. What do you mean? One of then returns List, and another one returns a string. We need both methods, because sometimes there are several messages. About getMessage() - I added it because you asked to

Reply:
I added it because you asked to. May we just delete it?

Reply:
Have added asserts to check that switch has message, successMessage, errorMessage

Reply:
Ok, let's leave it as is.
We'll refactor some park after we'll agree with interfaces